### DISCUSSION TITLE
[Feature] Import and display IPTC keywords stored in photo metadata

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8717] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have many photos with already specified keywords in IPTC:Keywords, so I suggest implementing the importing of them during photo upload. It’s a comma-separated list of keywords describing the content of the photo.

Those keywords should be searchable and visible in the details pane. Clicking on a tag should transition to search results or any other page that would show all photos with a specific keyword.

Another section with keywords should be also present on the “Explore” page.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Is this a duplicate of #3950?

**Reply by imler** [reply]
Well, it's related for sure, but he didn't specify exactly what kind of tags he wants to import, because they are stored not only in IPTC:Keywords, but also in other fields as well.

The second thing is that I specified additional info on what to do with those tags afterward.

It would be good to merge somehow those two proposals into one issue though :)

**Reply by bo0tzz** [reply]
I think the request I linked is very general and would include the tag you mentioned as well. Let's close this one, and feel free to elaborate on the other one :)
